sagemaker-inference-recommender/auto-scaling/optimize_endpoint_scaling.ipynb

```diff
@@ -652,7 +652,7 @@
     "    instance_type=instance_type,\n",
```

Comment:
<p>This is SageMaker V1 way of deleting endpoints - I would revert to the SageMaker V2 way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
As I don't use predicate to invoke. the predicate object was none, hence used the api to delete it.

Reply:
<p>But in step 8, you still deploy predictor</p><pre class="ql-syntax" spellcheck="false">predictor = model.deploy(
&nbsp;&nbsp;&nbsp; initial_instance_count=min_capacity, instance_type=instance_type, endpoint_name=endpoint_name
)
</pre><p>As such, it should be deleted with <code>predictor.delete_endpoint()</code> - we should stick with V2 SDK as that is best practice - deviating from such will confuse customers with multiple solutions </p>